src/sparse/impl/KokkosSparse_spgemm_mkl_impl.hpp

```diff
@@ -425,12 +424,6 @@ void mkl_symbolic(
      MKL\n"); return;
       }
   */
-  if (std::is_same<idx, int>::value) {
```

Comment:
I don't see what this if-block ends; is there a matching `}` for it? Should the if-block just be converted to a block -- `{`.

Reply:
@e10harvey there is not a matching brace to close this if block, that was what was causing the compilation error. There was an apparent copy+paste error in a past update to this file that duplicated the lines of code that I removed in this PR (and so removing the mismatch in open-closed braces)

Reply:
@e10harvey See lines 433-448 of this file which had mistaken duplication to lines 428-431, removed by this PR